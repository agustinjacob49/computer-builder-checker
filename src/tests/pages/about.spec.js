import { render } from '@testing-library/react';
import View from '../../pages/about/about';

describe('Home view', () => {
  test('Snapshot testing', () => {
    const { container } =  render(<View />);
    expect(container).toMatchSnapshot();
  });

});

