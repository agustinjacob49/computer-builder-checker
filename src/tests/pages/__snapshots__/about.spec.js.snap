// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home view Snapshot testing 1`] = `
<div>
  <p>
    Hi ! My name is Agustin Jacobsen and I develop this using a custom library to manage graphs and adjacency matrix
  </p>
</div>
`;
